flow/project.py

```diff
@@ -370,13 +370,16 @@ class SubmissionJobOperation(JobOperation):
     :type eligible_operations:
         list
     :param operations_with_unmet_preconditions:
-        A list of :py:class:`JobOperation` that will not be executed due to
-        unmet preconditions.
+        A list of :py:class:`JobOperation` that will not be executed in the
+        first pass of :meth:`FlowProject.run` due to unmet preconditions. These
```

Comment:
```suggestion
        current iteration of the :meth:`FlowProject.run` loop due to unmet preconditions. These
```

Reply:
These operations (unmet preconditions/met postconditions) are only seen on construction of the `SubmissionJobOperation`, so they are only relevant to the status of the data space on the first pass. After the first pass they are invalidated, and we depend entirely on the `run` loop to figure out what to run next, so the original version of this docstring is correct.